etc/profile-a-l/goldendict.profile

```diff
@@ -0,0 +1,62 @@
+# Firejail profile for goldendict
+# This file is overwritten after every install/update
+# Persistent local customizations
+include default.local
+# Persistent global definitions
+include globals.local
+
+noblacklist ${HOME}/.goldendict
+noblacklist ${HOME}/.cache/GoldenDict
+# Allow donwloading image from entries
+noblacklist ${DOWNLOADS}
+
+include disable-common.inc
+include disable-devel.inc
+include disable-exec.inc
+include disable-interpreters.inc
+include disable-passwdmgr.inc
+include disable-programs.inc
+include disable-shell.inc
+include disable-write-mnt.inc
```

Comment:
redundant due to `disable-mnt` below.

```suggestion
```

Reply:
removed disable-mnt

Reply:
So it needs read-only access to e.g. `/run/media/MY/MY-USB` but not to e.g. `~/Documents`?

Reply:
I changed my idea. Mounted directories should not be accessed at all. I'll add back `disable-mnt` and remove `disable-write-mnt.inc`.

`~/Documents` is also not needed, images will be downloaded to `~/Downloads`.